vars/all.yml

```diff
@@ -8,7 +8,7 @@ openshift_ansible_dir: "openshift-ansible/"
 openshift_playbook_path: "{{ openshift_ansible_dir }}/{{ 'playbooks/byo/config.yml' if kubevirt_openshift_version == '3.7' else 'playbooks/deploy_cluster.yml' }}"
 
 ### KubeVirt ###
-version: 0.12.0-alpha.2
+version: 0.12.0
```

Comment:
Could you please update also readmes?

There are many occurrences
```
[lbednar@lbednar kubevirt-ansible]$ git grep 0.12.0-alpha.2
glide.yaml:  version: v0.12.0-alpha.2
hack/run-tests.sh:tag=${DOCKER_TAG:-v0.12.0-alpha.2}
roles/kubevirt/README.md:| version |0.12.0-alpha.2|<ul><li>0.12.0-alpha.2</li><li>0.9.6</li><li>0.8.0</li><li>0.7.0</li><li>0.6.0</li><li>0.5.0</li><li>0.4.1</li><li>0.4.0</li><li>0.3.0</li><li>0.2.0</li><li>0.1.0</li></ul>|KubeVirt release version.|
roles/kubevirt/README.md:ansible-playbook -i inventory  -e version=0.12.0-alpha.2 -e apb_action=provision playbooks/kubevirt.yml
roles/kubevirt/defaults/main.yml:version: 0.12.0-alpha.2
roles/network-multus/defaults/main.yml:docker_tag: "v0.12.0-alpha.2"
tests/manifests/vm_with_sidecar_hook.yml:    hooks.kubevirt.io/hookSidecars: '[{"image": "kubevirt/example-hook-sidecar:v0.12.0-alpha.2"}]'
vars/README.md:| version |0.12.0-alpha.2|<ul><li>0.12.0-alpha.2</li><li>0.9.6</li><li>0.8.0</li><li>0.7.0</li><li>0.6.0</li><li>0.5.0</li><li>0.4.1</li><li>0.4.0</li><li>0.3.0</li><li>0.2.0</li><li>0.1.0</li></ul>|KubeVirt release version.|
vars/all.yml:version: 0.12.0-alpha.2
```

Reply:
Another PR is going to handle to upgrade to 0.13 https://github.com/kubevirt/kubevirt-ansible/pull/556